src/main/java/com/developers/hireasenior/controller/AccountController.java

```diff
@@ -0,0 +1,49 @@
+package com.developers.hireasenior.controller;
+
+import com.developers.hireasenior.dto.AccountDto;
+import com.developers.hireasenior.dto.request.TechnologyAndPeriodRequest;
+import com.developers.hireasenior.dto.response.ApiResponse;
+import com.developers.hireasenior.service.AccountService;
+import jakarta.validation.Valid;
+import lombok.AllArgsConstructor;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+@RestController
+@RequestMapping("/api/v1/account")
+@AllArgsConstructor
+public class AccountController {
+
+    private final AccountService accountService;
+
+    @GetMapping("/developersByTechnologiesAndAvailableTime")
+    public ResponseEntity<ApiResponse<List<AccountDto>>> filteredSeniorsTechnologyTime(@Valid @RequestBody TechnologyAndPeriodRequest technologyAndPeriodRequest) {
```

Comment:
@GetMappings actually cannot have a body. You can actually use a @PostMapping even if you are listing something. There is nothing wrong with it.